Earthfile

```diff
@@ -632,3 +632,7 @@ npm-update-all:
         RUN cd $nodepath && npm update
         SAVE ARTIFACT --if-exists $nodepath/package-lock.json AS LOCAL $nodepath/package-lock.json
     END
+
+export:
```

Comment:
Should this go under the `docs` subdir instead?

Reply:
> another potential idea is to just use `GIT CLONE github.com/earthly/earthly/docs .` in the external Earthfile.

Thanks Alex. I didn't realize that option. In this particular case the artifact reference is preferred for that external Earthfile since it is more versatile and can be parameterized to point to other artifacts not necessarily on GIT

Reply:
> Should this go under the `docs` subdir instead?

Changing it